src/NumericalAlgorithms/Spectral/SwshTransform.hpp

```diff
@@ -536,8 +535,7 @@ struct InverseSwshTransform<tmpl::list<TransformTags...>, Representation> {
   // file and friend requirements
   template <ComplexRepresentation FriendRepresentation, int FriendSpin,
             typename... CollocationThenCoefficientTypes, size_t... Is>
-  // NOLINTNEXTLINE(readability-redundant-declaratioon)
-  friend void detail::inverse_swsh_transform_impl(
+  friend void detail::inverse_swsh_transform_impl(  // NOLINT
```

Comment:
is there now multiple warnings for `redundant-declaration`?  If so we can disable one is SetupClangTidy.cmake and keep the one we no-linted before

Reply:
Oh! it's good you pointed this out. I'd switched to the trailing `NOLINT` because the previous version stopped working, but now that you point it out I realize it's because the previous version had a typo, so I'll just revert the change apart from fixing the typo.

Reply:
If you look at line 41 of `SetupClangTidy.cmake` we ignore a bunch of warnings already because they're either duplicates, wrong, or have too many false positives. Could you add the duplicate warning to that list?